backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java

```diff
@@ -94,6 +94,14 @@ public void createEntities() {
 
   @After
   public void removeEntities() {
+    for (ExperimentRun run : new ExperimentRun[] {experimentRun, experimentRun2}) {
+      DeleteExperimentRun deleteExperimentRun =
+          DeleteExperimentRun.newBuilder().setId(run.getId()).build();
+      DeleteExperimentRun.Response deleteExperimentRunResponse =
+          experimentRunServiceStub.deleteExperimentRun(deleteExperimentRun);
+      assertTrue(deleteExperimentRunResponse.getStatus());
+    }
+
```

Comment:
If we delete projet first then while deleting experiment_run we can not find the project while checking project validation and getting the error for project not found.